apps/challenges/aws_utils.py

```diff
@@ -0,0 +1,542 @@
+                    "value": "settings.{env}"
+                }},
+                {{
+                    "name": "CHALLENGE_PK",
+                    "value": "{challenge_pk}"
+                }},
+                {{
+                    "name": "CHALLENGE_QUEUE",
+                    "value": "{queue_name}"
+                }},
+                {{
+                  "name": "DEBUG",
+                  "value": "{DEBUG}"
+                }},
+                {{
+                  "name": "EMAIL_HOST",
+                  "value": "{EMAIL_HOST}"
+                }},
+                {{
+                  "name": "EMAIL_HOST_PASSWORD",
+                  "value": "{EMAIL_HOST_PASSWORD}"
+                }},
+                {{
+                  "name": "EMAIL_HOST_USER",
+                  "value": "{EMAIL_HOST_USER}"
+                }},
+                {{
+                  "name": "EMAIL_PORT",
+                  "value": "{EMAIL_PORT}"
+                }},
+                {{
+                  "name": "EMAIL_USE_TLS",
+                  "value": "{EMAIL_USE_TLS}"
+                }},
+                {{
+                  "name": "MEMCACHED_LOCATION",
+                  "value": "{MEMCACHED_LOCATION}"
+                }},
+                {{
+                  "name": "RDS_DB_NAME",
+                  "value": "{RDS_DB_NAME}"
+                }},
+                {{
+                  "name": "RDS_HOSTNAME",
+                  "value": "{RDS_HOSTNAME}"
+                }},
+                {{
+                  "name": "RDS_PASSWORD",
+                  "value": "{RDS_PASSWORD}"
+                }},
+                {{
+                  "name": "RDS_USERNAME",
+                  "value": "{RDS_USERNAME}"
+                }},
+                {{
+                  "name": "SECRET_KEY",
+                  "value": "{SECRET_KEY}"
+                }},
+                {{
+                  "name": "SENTRY_URL",
+                  "value": "{SENTRY_URL}"
+                }},
+            ],
+            "workingDirectory": "/code",
+            "readonlyRootFilesystem": False,
+            "logConfiguration": {{
+                "logDriver": "awslogs",
+                "options": {{
+                    "awslogs-group": "gsoc-2019-test",
+                    "awslogs-region": "us-east-1",
+                    "awslogs-stream-prefix": "{queue_name}",
+                }},
+            }},
+        }}
+    ],
+    "requiresCompatibilities":[
+        "FARGATE"
+    ],
+    "cpu": "1024",
+    "memory": "2048",
+}}
+"""
+
+service_definition = """
+{{
+    "cluster":"gsoc2019",
+    "serviceName":"{service_name}",
+    "taskDefinition":"{task_def_arn}",
+    "desiredCount":1,
+    "clientToken":"{client_token}",
+    "launchType":"FARGATE",
+    "platformVersion":"LATEST",
+    "networkConfiguration":{{
+        "awsvpcConfiguration": {{
+            "subnets": [
+                "subnet-e260d5be",
```

Comment:
Doesn't feel right to hard code the subnet-id here. If we want other people to use this code and run workers on their fargate cluster, it will be really painful for them. 

Ideally, all of this should come from a fargate config file or environment variables. We don't have to do this for this version but would be good to open an issue for this so that we keep an eye on this. (cc: @RishabhJain2018 @Ram81 )